ground/src/main/java/com/google/android/ground/ui/offlineareas/OfflineAreasFragment.kt

```diff
@@ -51,13 +59,24 @@ class OfflineAreasFragment : AbstractFragment() {
     val binding = OfflineAreasFragBinding.inflate(inflater, container, false)
     binding.viewModel = viewModel
     binding.lifecycleOwner = this
+    binding.offlineAreasListComposeView.setContent { AppTheme { ShowOfflineAreas() } }
 
     getAbstractActivity().setSupportActionBar(binding.offlineAreasToolbar)
 
-    val recyclerView = binding.offlineAreasList
-    recyclerView.setHasFixedSize(true)
-    recyclerView.layoutManager = LinearLayoutManager(context)
-    recyclerView.adapter = offlineAreaListAdapter
     return binding.root
   }
+
+  @Composable
+  private fun ShowOfflineAreas() {
```

Comment:
I am thinking to keep all composable functions outside fragments, we can have a separate file.

Reply:
We already have a separate file which contains the main logic for how the item looks.